nft_ingester/src/program_transformers/token_metadata/v1_asset.rs

```diff
@@ -344,71 +295,43 @@ pub async fn save_v1_asset<T: ConnectionTrait + TransactionTrait>(
-                    .add(asset_creators::Column::AssetId.eq(id.to_vec()))
-                    .add(asset_creators::Column::SlotUpdated.lt(slot_i)),
-            )
-            .all(conn)
-            .await?;
-        if existing_creators.len() > 0 {
-            let mut db_creators = Vec::with_capacity(creators.len());
-            for (i, c) in creators.into_iter().enumerate() {
-                if creators_set.contains(&c.address) {
-                    continue;
-                }
-                db_creators.push(asset_creators::ActiveModel {
-                    asset_id: Set(id.to_vec()),
-                    creator: Set(c.address.to_bytes().to_vec()),
-                    share: Set(c.share as i32),
-                    verified: Set(c.verified),
-                    seq: Set(0), // do we need this here @micheal-danenberg?
-                    slot_updated: Set(slot_i),
-                    position: Set(i as i16),
-                    ..Default::default()
-                });
-                creators_set.insert(c.address);
-            }
-            let txn = conn.begin().await?;
-            asset_creators::Entity::delete_many()
-                .filter(
-                    Condition::all()
-                        .add(asset_creators::Column::AssetId.eq(id.to_vec()))
-                        .add(asset_creators::Column::SlotUpdated.lt(slot_i)),
-                )
-                .exec(&txn)
-                .await?;
-            if db_creators.len() > 0 {
-                let mut query = asset_creators::Entity::insert_many(db_creators)
-                    .on_conflict(
-                        OnConflict::columns([
-                            asset_creators::Column::AssetId,
-                            asset_creators::Column::Position,
-                        ])
-                        .update_columns([
-                            asset_creators::Column::Creator,
-                            asset_creators::Column::Share,
-                            asset_creators::Column::Verified,
-                            asset_creators::Column::Seq,
-                            asset_creators::Column::SlotUpdated,
-                        ])
-                        .to_owned(),
-                    )
-                    .build(DbBackend::Postgres);
-                query.sql = format!(
-                    "{} WHERE excluded.slot_updated > asset_creators.slot_updated",
-                    query.sql
-                );
-                txn.execute(query).await?;
-            }
-            txn.commit().await?;
+        let mut db_creators = Vec::with_capacity(creators.len());
+        for (i, c) in creators.into_iter().enumerate() {
+            db_creators.push(asset_creators::ActiveModel {
+                asset_id: Set(id.to_vec()),
+                creator: Set(c.address.to_bytes().to_vec()),
+                share: Set(c.share as i32),
+                verified: Set(c.verified),
+                seq: Set(0),
```

Comment:
I would keep the comment here. I'm pretty sure you don't need seq. You either use it or you don't use it at all imo, but keeping at 0 with a comment saying someting like "may not be necessary" is fine for now